CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+
+### b. Conflicts of Interest
+
+In the event of a conflict of interest with any COC committee member, the relevant member will recuse themselves from handling the report. Involved parties are never part of the review process.
+
+If you are concerned about submitting a report that will reach any specific individual, please contact one of the COC committee members or Quansight Labs Co-Directors directly.
+
+### c. What to Include in a Report
+
+Our ability to address any Code of Conduct incident in a timely and effective manner is critically dependent on the amount & the precision of all information provided. Please include as much of the following as possible in an incident report:
+
++ Your contact information (including your preferred medium of contact).
+    + If you wish to remain anonymous, your personal contact details will not be shared beyond
+      the person receiving the initial report.
++ Approximate time & location of the incident (be as specific as possible).
++ Identifying information of all individuals actively involved in the COC incident.
+    + This may include names, nicknames/pseudonyms, usernames/screen-names, physical
+  descriptions, etc.
++ Witnesses (if any)
+    + Provide names and contact details if available
++ Description of the circumstances/context surrounding the incident (e.g., at a conference, in a private meeting, during an online chat, etc.)
++ Description of the behavior
+    + If reporting harassing language, be specific about the words used.
+    + Provide a detailed account of what happened.
+    + Provide any available supporting records (e.g. email, screenshots, links to public GitHub
+    issues, mailing list archives, or public chat logs, etc.)
++ State whether you believe this incident is ongoing and/or part of an ongoing pattern of
+  behavior.
++ Any additional information that may be helpful.
+
+### d. Procedures after Filing a Report
+
+**Reviewing the Report**
+
+The Quansight COC committee will acknowledge receipt of the incident report within one or two business days. They will make all efforts to review the incident report within a week to determine:
+
+* whether this is an ongoing situation;
+* whether there is a threat to anyone’s physical safety;
+* what specfically happened;
+* whether this incident does constitute an infraction of the Code of Conduct; and
+* who the bad actors were, if any.
+
+All reports will be reviewed and will result in a response necessary and appropriate to the circumstances. Where additional perspectives are needed, the team may seek insight from others with relevant expertise or experience. The confidentiality of the incident's reporter will be kept at all times.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately. If any individual engages in unacceptable behavior again, the COC committee may reccommend any action they deem appropriate (as enumerated below).
```

Comment:
This is odd. Consequences can also follow after a first incident, not only after a repeat. It's also unrelated to the `Reviewing the Report` heading of this section - I suggest to remove it.